docs/howto/openstack/neutron/del-network.md

```diff
@@ -0,0 +1,553 @@
+    the following:
+    
+    ```bash
+    openstack network list --internal
+    ```
+    ```plain
+    +--------------------------------------+--------------+--------------------------------------+
+    | ID                                   | Name         | Subnets                              |
+    +--------------------------------------+--------------+--------------------------------------+
+    | 1f94d315-7ca1-4d44-acc1-09c6c650df74 | mayo         |                                      |
+    | 9b127d2c-01d7-4803-994f-f88292870c1d | teslin       | bd1d0ff2-7270-4a9a-a7ad-fff47e997e7b |
+    | cb0a298a-bbb6-4ad6-832a-1456dafe45db | carmacks     | 7fa9e5a2-7d5a-466e-b120-7d2bffb99ce5 |
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+    
+Let us assume you wish to delete the network named `carmacks`.
+
+## Determining component dependencies
+
+If the network to be deleted has a subnet component --- and most likely 
+it will have ---, you will first have to delete the subnet before 
+deleting the network. If, in addition, the network is behind a router 
+(figurately speaking), then before deleting the subnet, you will have 
+to disconnect it from the router. Finally, you will have the option to 
+delete the router also. Let us see what the situation is with network 
+`carmacks`.
+
+=== "{{gui}}"
+    For more information on `carmacks`, click the three-dot icon 
+    (right-hand side of the network row) and select _View details_. Four 
+    tabs immediately appear below; _Details_, _Ports_, _Subnets_, and 
+    _Routers_. Looking at the _Details_ tab, it is clear that network 
+    `carmacks` has a subnet and is behind a router. You may click on tabs 
+    _Subnets_ and _Routers_, to see more information regarding the network 
+    subnet and the router in front of the network.
+    
+    ![Network details](assets/del-net/shot-02.png)
+=== "OpenStack CLI"
+    To quickly check whether network `carmacks` has a subnet or not, type:
+        
+    ```bash
+    openstack network show carmacks -c subnets
+    ```
+    ```plain
+    +---------+--------------------------------------+
+    | Field   | Value                                |
+    +---------+--------------------------------------+
+    | subnets | 7fa9e5a2-7d5a-466e-b120-7d2bffb99ce5 |
+    +---------+--------------------------------------+
+    ```
+        
+    If the value for the field `subnets` is non-empty, like in the example 
+    output above, that means the network has a subnet indeed, and the value 
+    is the ID of that subnet. At this point, it helps to assign the subnet 
+    ID to an environment variable, like so:
+        
+    ```bash
+    SUBNET_ID="7fa9e5a2-7d5a-466e-b120-7d2bffb99ce5"
+    ```
+        
+    What about a router in front of `carmacks`? You might think to check 
```

Comment:
Maybe it's just me, but "you might think to check" sounds oddly condescending to me. Maybe use "you might try checking" instead?

Reply:
That is also done!